src/aspire/utils/coor_trans.py

```diff
@@ -7,7 +7,7 @@
 from scipy.linalg import svd
 
 from aspire.utils import ensure
-from aspire.utils.matlab_compat import Random, randn
+from aspire.utils.random import Random, randn, random
```

Comment:
`random` unused.

Reply:
yes, right, only in comment.  1ac4268b509c5064f1a47a51ed569f4bf32b04ee